lib/v2/gov/radar.js

```diff
@@ -305,6 +305,17 @@ module.exports = {
             },
         ],
     },
+    'wsjkw.sh.gov.cn': {
+        _name: '上海卫健委',
+        '.': [
+            {
+                title: '疫情通报',
```

Comment:
```suggestion
    'sh.gov.cn': {
        _name: '上海市人民政府',
        'wsjkw': [
            {
                title: '上海卫健委 疫情通报',
```
Please also combine the above item "上海市职业能力考试院 考试项目" into this rules